docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,305 @@
+    
+    ```{.bash data-prompt="$" }
+    $ kubectl create namespace monitoring-system
+    ```
+
+5. Create the Secrets object using the YAML file you created previously. Replace the `<filename>` placeholder with your value.
+
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f pmm-api-vmoperator.yaml -n <namespace>
+    ```
+
+6. Check that the secret is created. The following command checks the secret for the resource named `pmm-token-vmoperator` (as defined in the `metadata.name` option in the secrets file). If you defined another resource name, specify your value.  
+
+   ```{.bash data-prompt="$" }
+   $ kubectl get secret pmm-token-vmoperator -n <namespace>
+   ```
+
+### Create a ConfigMap to mount for kube-state-metrics 
+
+The `kube-state-metrics` (KSM) is a simple service that listens to the Kubernetes API server and generates metrics about the state of various objects - Pods, Deployments, Services and Custom Resources. 
+
+To instruct the `kube-state-metrics` what metrics to capture, create the ConfigMap and mount it to a container. 
+
+1. Edit the [example `configmap.yaml` configuration file](https://raw.githubusercontent.com/percona/percona-everest-cli/main/data/crds/victoriametrics/kube-state-metrics/configmap.yaml) and specify the `<namespace>`. The Namespace must match the Namespace where you created the Secret. 
+
+2. Apply the configuration
+
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f <path-to-configmap.yaml>
+    ```
+
+    As a result, you have the `customresource-config-ksm` ConfigMap created. 
+
+### Install the Victoria Metrics Operator Helm chart
+
+1. Add the dependency repositories of [victoria-metrics-k8s-stack](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-k8s-stack) chart. 
+
+    ```{.bash data-prompt="$" }
+    $ helm repo add grafana https://grafana.github.io/helm-charts
+    $ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    ```
+
+2. Update the repositories.
+
+    ```{.bash data-prompt="$" }
+    $ helm repo update
+    ```
+
+3. Add the Victoria Metrics Operator repository.
+
+    ```{.bash data-prompt="$" }
+    $ helm repo add vm https://victoriametrics.github.io/helm-charts/
+    $ helm repo update
+    ```
+
+4. Export default values of `victoria-metrics-k8s-stack` chart to file `values.yaml` file.
+
+    ```{.bash data-prompt="$" }
+    $ helm show values vm/victoria-metrics-k8s-stack > values.yaml
+    ```
+
+5. Edit the `values.yaml` file and specify the following:
+
+    * the IP address / hostname of the PMM server in the `externalVM.write.url` option
+    * specify the unique name or an ID of the Kubernetes cluster in the ``vmagent.spec.externalLabels.cluster` option. Ensure to set different values if you are sending metrics from multiple Kubernetes clusters to the same PMM Server.
```

Comment:
The field is now renamed to `k8s_cluster_id`. Sorry for the late changes.

Reply:
Updated